src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -2,25 +2,33 @@ package racing.domain
 
 private const val MINIMUM_COUNT_OF_CARS = 2
 
-class Cars(private val cars: List<Car>) {
+private const val DELIMITER_OF_NAME = ","
+
+class Cars(private val value: List<Car>) {
 
     init {
-        require(cars.size >= MINIMUM_COUNT_OF_CARS) {
+        require(value.size >= MINIMUM_COUNT_OF_CARS) {
             "자동차는 최소 ${MINIMUM_COUNT_OF_CARS}대 이상이어야 합니다."
         }
     }
 
-    constructor(numberOfCars: Int) : this(
-        (1..numberOfCars).map { Car() }
-    )
+    companion object {
+        fun create(names: String): Cars {
+            return Cars(names.trim().split(DELIMITER_OF_NAME).map { Car.from(it.trim()) })
```

Comment:
너무 긴 코드는 적절히 변수로 추출해서 나눠주는 것이 가독성이 더 좋을 것 같아요. 😃
```suggestion
            val cars = names.trim().split(DELIMITER_OF_NAME)
                .map { Car.from(it.trim()) }
            return Cars(cars)
```